Add support for NovaDigital Switch (single/quad)

xgustavoh opened the pull request:
Add support for [NovaDigital Switch](https://www.novadigitalsmart.com.br/produtos/interruptor-touch-wi-fi-ws-us)

- Single:
<img width="337" alt="image" src="https://github.com/make-all/tuya-local/assets/19254287/e7250cab-d3dc-4d2f-aeac-df1bdbd3be79">


- Quad:
<img width="349" alt="image" src="https://github.com/make-all/tuya-local/assets/19254287/d925bd55-a9c5-4d38-8f27-6078d15781fd">

make-all commented:
I have merged the single switch into somgom_single_switch, which is identical (dp 14 was missing from the config, but that is because it did not show up in logs, and the optional dps support was not available yet when it was originally added).

Please resubmit only the quad switch (and no .DS_Store file - you should add that to your global .gitignore if your OS automatically generates such files all over the place).

make-all reviewed (changes requested):
As above, please resubmit just the quad switch config.

xgustavoh commented:
@make-all 

Perfect!
I rebased it with just the `Novadigital quad switch` and removed the `.DS_Store`.

--

Thanks a lot for the global .gitignore tip.
